governance/polls/Reserve Governance Facilitator Apppointment - August 21, 2023.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Reserve Governance Facilitator Apppointment - August 21, 2023
+summary: Signal your support or opposition to appointing the following candidates as Reserve Governance Facilitator(s).
+discussion_link: https://forum.makerdao.com/t/poll-notification-selection-of-reserve-governance-facilitators/21671
+parameters:
+  input_format: 
+    type: choose-free
+    options: [1,2,3,4]
```

Comment:
Not sure if the behaviour you've described is intended. If it isn't, it's a bit sketchy to rely on it.